plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/messages.properties

```diff
@@ -316,8 +316,8 @@ CLF_SPEC_WRONG_TYPE=error;;;Annotation @Spec may only be used with formal parame
 CLF_SPEC_WRONG_ADD_MEMBERTYPE=error;;;Type of structural member {0} of spec parameter must be a subtype of {1}: {2}.
 # no parameters required
 CLF_SPEC_MULTIPLE=error;;;Only a single formal parameter in each constructor may be annotated with @Spec.
-# 0: name of the superfluous property
-CLF_SPEC_SUPERFLUOUS_PROPERTIES=warning;;;{0} is not used in constructor.
+# 0: name of the superfluous property, 1: the type not defining the property
+CLF_SPEC_SUPERFLUOUS_PROPERTIES=warning;;;{0} is not defined in {1}.
```

Comment:
Just an idea: would it make sense to keep the stronger message in case of spec ctors to highlight that the generated constructor won't initialize anything for that property in the newly created instance?

Reply:
I'm not quite sure whether it is "stronger". I actually found some cases where I though: Ah! It this or that type... just the "not used in constructor" is not really a hint.

Reply:
ok, then leave as is. And as a plus: it's one special case less! 😄